tables.md

```diff
@@ -0,0 +1,204 @@
+| Year        | 2023        | Liverpool         | 5.3         | billion | U.S dollar |
+| Year        | 2023        | Manchester City   | 5           | billion | U.S dollar |
+| Year        | 2023        | Chelsea           | 3.1         | billion | U.S dollar |
+| Year        | 2023        | Tottenham Hotspur | 2.8         | billion | U.S dollar |
+
+The second one is american_football.csv.
+
+| period_type | period_code | variable_name        | observation | measure | unit       |
+| ----------- | ----------- | -------------------- | ----------- | ------- | ---------- |
+| Year        | 2023        | Dallas Cowboys       | 9           | billion | U.S dollar |
+| Year        | 2023        | New England Patriots | 7           | billion | U.S dollar |
+| Year        | 2023        | Los Angeles Rams     | 6.9         | billion | U.S dollar |
+| Year        | 2023        | New York Giants      | 6.8         | billion | U.S dollar |
+| Year        | 2023        | Chicago Bears        | 6.3         | billion | U.S dollar |
+
+There are additional descriptors for human readability that you can add to your column sections. These include:
+
+- `suppressOutput`  If true, suppresses any output that would be generated when converting cells in this column. The value of this property becomes the suppress output annotation for the described column. The default is false.(from w.3/org-tabular-metadata)
+-  `propertyuri`, `valueurl`, `aboutUrl`  This metadata creates a relationship model between data in each column by different combinations of aboutUrl, propertyUrl, and valueUrl on existing columns, and defining new virtual columns to supply additional information. (from w.3/org-tabular-metadata)
+- `label` this can be used for additional context.
+
+
+## Table Breakdown
+
+This section will help you how to write your table section of your JSON-LD file. But also, how to describe each column from your csv file. 
+
+The first section will show you what needs to be done for each csv you are using.
+
+The second section will go through a step by step process of writing and describing the columns in your csv.
+
+## First Section
+
+```JSON
+"tables": [
+    {
+        "url": "football.csv",
+        "tableSchema": {
+            "columns": [
+```                
+                
+In order to write a JSON-LD to include mulitple csvs. You need to include the csvs column details within a tables [ ] section. 
+
+`url` This link property gives the single URL of the CSV file that the table is held in, relative to the location of the metadata document. (from w.3/org-tabular-metadata)
+
+`tableSchema` An object property that provides a single schema description, used as the default for all the tables in the group. This may be provided as an embedded object within the JSON metadata or as a URL reference to a separate JSON object that is a schema description.(from w.3/org-tabular-metadata)
+
```

Comment:
url needs fixing

Reply:
It was only used as an example of where that text has come from. Do we need to reference this?